recipes/phytop/src/Astral.py

```
@@ -0,0 +1,590 @@
+import sys, os
+import re
+import math
```

Ruff

`math` imported but unused

Remove unused import: `math`

(F401)